priv/cabbage/Makefile

```diff
@@ -6,7 +6,7 @@ test:
 	mix test test/itest
 
 clean:
-	docker-compose down && docker rm -f $$(docker ps -a -q) && docker volume rm $$(docker volume ls -q)
+	docker-compose down && docker volume prune --force
```

Comment:
The `docker ...` were wiping everything on the host machine not limited to the docker-compose definition